jsk_unitree_robot/jsk_unitree_startup/scripts/emotion_talker.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import rospy
+from std_msgs.msg import String
+from speech_recognition_msgs.msg import SpeechRecognitionCandidates
+from jsk_recognition_msgs.msg import PeoplePoseArray
+from dialogflow_task_executive.msg import DialogResponse
+import time
+import message_filters
+
+from threading import Lock
+
+
+class testNode():
+    def __init__(self):
+        # Publisher
+        self.pub = rospy.Publisher('/emotion', String, queue_size=1)
+
+        # Subscriber
+        self.sub1 = rospy.Subscriber('/speech_to_text', SpeechRecognitionCandidates, self.speech_callback)
+        self.sub2 = rospy.Subscriber('/people_pose', PeoplePoseArray, callback=self.callback)
+        self.sub3 = rospy.Subscriber('/dialog_response', DialogResponse, callback=self.df_cb)
```

Comment:
@iory @mqcmd196 please write system diaglam, specially dialogflow, speech recognition and chat
@a-ichikura please add emotino part in their diaglam